docs/application/native/guides/internationalization/i18n.md

```diff
@@ -871,9 +871,9 @@ The number-minus-offset value is formatted using a Number Format for the Plural
 
 ### Defining Custom Plural Rules:
 
-If you need to use Plural Format with custom rules, you can create a Plural Rules object
-and pass it to Plural Format's constructor. If you also specify a locale in this constructor,
-this locale will be used to format the number in the message texts.
+To add custom plural rules to Plural Format, create a Plural Rules object and put it in
+Plural Format's constructor. If you also pass a locale object to constructor, this locale
+will affect formatted number in the returned message texts.
 
```

Comment:
Please change as:

To add custom plural rules to the PluralFormat, create a Plural Rules object and add it in Plural Format constructor. If you also pass a locale object to constructor, this locale will affect formatted number in the returned message texts.